driver/compile_common.ml

```diff
@@ -121,7 +121,8 @@ let implementation info ~backend =
     let parsed = parse_impl info in
     if Clflags.(should_stop_after Compiler_pass.Parsing) then () else begin
       let typed = typecheck_impl info parsed in
-      if Clflags.(should_stop_after Compiler_pass.Typing) then () else begin
+      if Clflags.(should_stop_after Compiler_pass.Typing)
+      then () else begin
```

Comment:
Would you mind removing this formatting change to reduce the number of files touched by this PR?

Reply:
sorry, I shouldn't use patdiff on it. fixed.